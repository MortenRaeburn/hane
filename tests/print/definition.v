Definition id (A : Set) (x : A) : A := x.
Print id.
