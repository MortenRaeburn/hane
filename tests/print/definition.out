Definition id : forall[Set] (forall['0] ('1)) := fun[Set] (fun['0] ('0)).
